pkg/services/object/get/exec.go

```diff
@@ -198,6 +206,61 @@ func (exec *execCtx) getChild(id oid.ID, rng *objectSDK.Range, withHdr bool) (*o
 	return child, ok
 }
 
+func (exec *execCtx) getChildStream(id oid.ID, rng *objectSDK.Range, withHdr bool) (*objectSDK.Object, io.ReadCloser, bool) {
```

Comment:
i was surprised to see a pipe. But it seems reasonable if we're limited to the declared interface. But according to the existing use cases, it can be avoided: the result is only needed to copy to the original stream. Therefore, this could be
```suggestion
func (exec *execCtx) copyChild(dst ObjectWriter, id oid.ID, rng *objectSDK.Range, withHdr bool) bool {
```

Reply:
I tried to do it. Is that what you meant?